MANIFEST.in

```diff
@@ -1,6 +1,6 @@
 
 recursive-include Orange *.pyx *.py *.c *.cpp README* LICENSE
-recursive-include Orange/datasets *.tab *.basket *.info
+recursive-include Orange/datasets *.tab *.basket *.info *.dst
```

Comment:
Would you also add .csv, .metadata, and any other reasonable format we support and might ship in the future?